webpack.config.js

```diff
@@ -52,10 +53,15 @@ module.exports = {
             loader: 'json-loader'
         }]
     },
-    postcss: [autoprefixer],
+    postcss: [
+        autoprefixer({
+            browsers: ['last 3 versions', 'Safari >= 8', 'iOS >= 8']
```

Comment:
You're right that we probably should standardize on a version of autoprefixer across all the repos that use it. In addition to that, we should probably also define an explicit browser target rather than just "whatever `last 3 versions` means right now".